src/SFML/Window/Android/WindowImplAndroid.cpp

```diff
@@ -462,28 +462,20 @@ int WindowImplAndroid::processKeyEvent(AInputEvent* inputEvent, ActivityStates&
 ////////////////////////////////////////////////////////////
 int WindowImplAndroid::processMotionEvent(AInputEvent* inputEvent, ActivityStates& states)
 {
-    const std::int32_t device = AInputEvent_getSource(inputEvent);
-
-    Event event;
-
-    if (device == AINPUT_SOURCE_MOUSE)
-        event = Event::MouseMoved{};
-    else if (static_cast<std::uint32_t>(device) & AINPUT_SOURCE_TOUCHSCREEN)
-        event = Event::TouchMoved{};
```

Comment:
How is it that we can get rid of this?

Reply:
That code did nothing, it's a no-op -- the same conditions are checked below when `forwardEvent` is called. I believe my change is equivalent in behavior to the previous code.